charts/istio-authorizer/values.yaml

```diff
@@ -0,0 +1,81 @@
+## istio-authorizer image parameters
+##
+image:
+  ## Image repository
+  ##
+  repository: acp.artifactory.cloudentity.com/istio-authorizer
+
+  ## Image pull policy
+  ##
+  pullPolicy: IfNotPresent
+
+  ## Image tag (immutable tags are recommended, default is the chart appVersion)
+  ##
+  tag: ""
+
+## istio-authorizer image registry secret names as an array
+##
+imagePullSecrets:
+  - name: artifactory
+
+## String to partially override istio-authorizer.name
+##
+#nameOverride: ""
+
+## String to fully override istio-authorizer.fullname
+##
+# fullnameOverride: ""
+
+## Service account to use by the istio-authorizer
+##
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+## Issuer URL
```

Comment:
Let's add a bit more description to this field.
What should the issuerURL point to etc.

Reply:
The `issuerURL` is used as the value of the `iss` claim in access tokens that identifies the principal that issued the JSON Web Token (JWT). It is a case-sensitive string that should contain StringOrURI value. Using the `issuerURL` parameter is OPTIONAL.

Would that be ok? That's all I know about it :D